src/dfx/src/lib/network/id.rs

```diff
@@ -0,0 +1,24 @@
+use crate::lib::error::DfxResult;
+use crate::lib::network::local_server_descriptor::LocalServerDescriptor;
+
+use anyhow::Context;
+use fn_error_context::context;
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+
+#[derive(Deserialize, Serialize, Debug)]
+struct Id {
+    created: OffsetDateTime,
+}
```

Comment:
```suggestion
#[derive(Deserialize, Serialize, Debug)]
struct NetworkMetadata {
    created: OffsetDateTime,
}
```
This has a confusing name. How can a struct that only contains a timestamp be an identifier?

Reply:
Thanks!